lib/classes/tile.js

```diff
@@ -86,7 +87,8 @@ GridGame.classes.tile = function (x, y) {
     .removeClass(function(index, css) {
       return (css.match (/\btile_\S+/g) || []).join(' ');
     })
-    .toggleClass('tile_city', this.city);
+    .toggleClass('tile_city', this.city)
+      .toggleClass('tile_rock', this.rock);
```

Comment:
Minor thing: it would be better if you dedented the final line to align with the rest of the lines. Easier to read - I suspect this is just whitespace added by an editor?